scripts/update-cards.js

```diff
@@ -77,9 +79,8 @@ const updateMinionsJSFile = (cardsArray) => {
         if (text && !type) {
             synergies = text.match(synergyRegex) || [];
 
-            // Check for Lightfang
-            const isLightFang = text.match(/each minion type/gi);
-            if (isLightFang) {
+            const isMenagerie = text.match(/each[\r\n\s]*minion type|different[\r\n\s]*minion type/gi);
```

Comment:
Was there a card added with the text: "different minion type"? Aggem can be matched with "each minion type" and it will probably be excluded when quilboars aren't in a game.

Reply:
this block is only checked for minions with no type, so it shouldn't add all types to that quilboar.
I added this extra regex for consistency so it will catch the mug and jug.

Reply:
👍 I'll merge this in once the patch is released.